packages/mds-geography-service/service/validators.ts

```diff
@@ -58,14 +58,14 @@ export const { validate: validateGeographyDomainCreateModel, isValid: isValidGeo
         name: { type: 'string', maxLength: 255, nullable: true, default: null },
         description: { type: 'string', maxLength: 255, nullable: true, default: null },
         effective_date: { type: 'integer', nullable: true, default: null },
-        publish_date: { type: 'integer', nullable: true, default: null },
```

Comment:
Why removed?

Reply:
because you cant create a geography and set a publish_date at the same time.